docs/content/doc/usage/issue-references.en-us.md

```diff
@@ -0,0 +1,149 @@
+---
+date: "2019-11-21T17:00:00-03:00"
+title: "Usage: Automatically Linked References"
+slug: "automatically-linked-references"
+weight: 15
+toc: true
+draft: false
+menu:
+  sidebar:
+    parent: "usage"
+    name: "Automatically Linked References"
+    weight: 15
+    identifier: "automatically-linked-references"
+---
+
+# Automatically Linked References in Issues, Pull Requests and Commit Messages
+
+When an issue, pull request or comment is posted, the text description is parsed
+in search for references. These references will be shown as links in the Issue View
+and, in some cases, produce certain _actions_.
+
+Likewise, commit messages are parsed when they are listed, and _actions_
+are can be triggered when they are pushed to the main branch.
+
+To prevent the creation of unintended references, there are certain rules
+for them to be recognized. For example, they should not be included inside code
+text. They should also be reasonably cleared from their surrounding text
+(for example, using spaces).
+
+## User Mentions
+
+When a text in the form `@username` is found and `username` matches the name
+of an existing user, a _mention_ reference is created. This will be shown
```

Comment:
Orgnization name and team name?

Reply:
> 
> 
> Orgnization name and team name?

@lunny In fact references for organizations and teams are completely broken:

**Organizations**: No notification is created. Link renders OK.
**Teams**: No notification is created. Link resolves to an invalid page.

I'll have to create a PR for that.

Reply:
On a second thought, skipping the organization in the mention notifications is explicit. May be it should remain that way?

Reply:
@lunny Update the doc. I'll work on the team link rendering problem, but it should be a separate PR.

Reply:
#9127 created to fix the `@team` mentions.

Reply:
Sorry, bad copy and paste:

**Organizations**: No notification is created. Link renders OK.
**Teams**: Notifications _are sent_. Link resolves to an invalid page. See #9127

Reply:
I believe that not mailing the whole organization on an `@org` reference was intentional. `@team` reference fixed in #9127. Docs fixed as well.